packages/go_router/lib/src/configuration.dart

```diff
@@ -552,7 +552,7 @@ class RouteConfiguration {
         final String fullPath = concatenatePaths(parentFullpath, route.path);
         sb.writeln('  => ${''.padLeft(depth * 2)}$fullPath');
         _debugFullPathsFor(route.routes, fullPath, depth + 1, sb);
-      } else if (route is ShellRoute) {
+      } else if (route is ShellRoute || route is StatefulShellRoute) {
```

Comment:
```suggestion
      } else if (route is ShellRouteBase) {
```

Reply:
I'm pushing changes as you suggested.

Reply:
It should be fine as long as they follow the same contract of ShellRouteBase.